baselines/dom.generated.d.ts

```diff
@@ -3796,7 +3796,12 @@ declare var FocusNavigationEvent: {
 }
 
 interface FormData {
-    append(name: any, value: any, blobName?: string): void;
+    append(name: any, value: any, fileName?: string): void;
+    delete(name: any): void;
+    get(name: any): any;
+    getAll(name: any): any;
+    has(name: any): any;
+    set(name: any, value: any, fileName?: string): void;
 }
```

Comment:
From the spec: https://xhr.spec.whatwg.org/#formdata the definition should look like:
```ts
type FormDataEntryValue = File or string;

interface FormData {
  append(name: string, value: string): void;
  append(name:string, blobValue:Blob, filename?:string):void;
  delete(name: string):void;
  get(name:string):  FormDataEntryValue | null;
  getAll(name: string): FormDataEntryValue[];
  has(name: string): boolean;
  set(name:string, value:string): void;
  set(name: string, blobValue:Blob, filename?: string): void;
};
```